openslides/core/models.py

```diff
@@ -82,6 +82,14 @@ class Projector(RESTModelMixin, models.Model):
     width = models.PositiveIntegerField(default=1024)
     height = models.PositiveIntegerField(default=768)
 
+    background_color = models.CharField(max_length=7, default="#ffffff")
+    header_background_color = models.CharField(max_length=7, default="#317796")
+    header_font_color = models.CharField(max_length=7, default="#f5f5f5")
+    header_h1_color = models.CharField(max_length=7, default="#317796")
+    show_header_footer = models.BooleanField(default=True)
+    show_title = models.BooleanField(default=True)
+    show_logo = models.BooleanField(default=True)
```

Comment:
@FinnStutzenstein I think about your changes and I don't like to move all projector config options to projector model.
The following options should we keep as global config options (they should be managed on the settings for all projectors):
- header_background_color
- header_font_color
- header_h1_color
Also very useful is to reset to default value which is available for config options only.

Only the following options are useful as projector model options because it is useful to manage these options separately for each projector:
- background_color
- show_header_footer
- show_title
- show_logo

Can you please change this in your pull request and updated the migration file? Thanks.

Reply:
As discussed with @normanjaeckel and @FinnStutzenstein we can merge this